manifest.json

```diff
@@ -0,0 +1,19 @@
+ {
```

Comment:
If you use "chrome_url_overrides" that will give you the ability to change the new tab as you like